WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsListView.swift

```diff
@@ -58,8 +58,9 @@ private extension CustomFieldsListHostingController {
     }
 
     @objc func saveCustomField() {
-        // todo: call viewmodel's save function
-        navigationController?.popViewController(animated: true)
```

Comment:
I removed this to keep the behavior aligned with Android, we keep the user on the screen after saving.

Reply:
Sounds good, I forgot to mention this navigation was just to test the new navigation functionality in my PR.